src/hooks/useClipboard.tsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from "react";
+
+interface UseClipboardOptions {
+  timeout?: number;
+}
+
+export const useClipboard = (options: UseClipboardOptions = {}) => {
+  const [copied, setCopied] = useState<Boolean>(false);
+
+  const onCopy = (text: string) => {
+    navigator.clipboard.writeText(text);
```

Comment:
In this case it won't make any practical difference as it runs instantly, but these clipboard methods return a promise so for completeness maybe call `setCopied` and the timeout within `.then()`?